pkg/kubectl/cmd/run.go

```diff
@@ -34,7 +35,7 @@ import (
 
 const (
 	run_long = `Create and run a particular image, possibly replicated.
-Creates a replication controller to manage the created container(s).`
+Creates a deployment or job to manage the created container(s).`
```

Comment:
I see all the previous generators in place (factory.go), just adding the new ones and convincing people to use the new ones. I'm perfectly ok with this approach.